src/components/map/MapView.js

```diff
@@ -1,39 +1,49 @@
 import React, { Component } from 'react';
-import Mapbox, { Layer } from 'react-mapbox-gl';
-import OpenSeatMarker from './OpenSeatMarker';
-import OpenSeatPopup from './OpenSeatPopup';
+import Mapbox, { Layer, Feature } from 'react-mapbox-gl';
+import DivisionPopup from './DivisionPopup';
 import { openSeatsPropType } from './propTypes';
 import divisionData from '../../data/divisions.json';
 
 const ZOOM = 10;
 const CENTER = [-75.1452, 39.9826];
-const LAYER_OPTIONS = {
-  source: {
-    type: 'geojson',
-    data: divisionData
-  }
+
+// TODO: This function should return only the open
+// seats in the specified division.
+const getOpenSeats = (division, openSeats) => {
```

Comment:
This will need to be implemented. You can get the properties from the divison, which will include the division number.

Here's some pseudocode:

```
const divisionNumber = division.properties['DIVISON_NUM'];
return openSeats.filter(seat => seat.divisionNumber === divisionNumber);
```